hack/ocm-minikube-ramen.sh

```diff
@@ -78,6 +78,16 @@ ramen_deploy()
 	kube_context_set_undo
 	kubectl --context ${2} -n ramen-system wait deployments --all --for condition=available --timeout 60s
 	# Add s3 profile to ramen config
+	cat <<EOF | kubectl --context ${2} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3secret
+  namespace: ramen-system
+stringData:
+  AWS_ACCESS_KEY_ID: "minio"
+  AWS_SECRET_ACCESS_KEY: "minio123"
+EOF
```

Comment:
(nit) I think https://github.com/RamenDR/ocm-ramen-samples/blob/main/subscriptions/busybox/s3secret.yaml can be deleted since s3secret is now part of ramen not the application.

Reply:
Ack! will do once this is merged.

Reply:
Opened a PR in samples for the same [here](https://github.com/RamenDR/ocm-ramen-samples/pull/8)